scheduler/routebroadcastscheduler_test.go

```diff
@@ -32,6 +34,7 @@ var _ = Describe("RouteBroadcastScheduler", func() {
 		Context(prefix, func() {
 			BeforeEach(func() {
 				natsClient = diegonats.NewFakeClient()
+				cfg.JitterFactor = 0.2
```

Comment:
this won't work. you've initialized`cfg` as a pointer to config.RouteEmitterConfig in line 25 but never gave it any value to point to. so this test will throw a nil ref panic. please run your tests first before requesting a review.

I've added a suggestion that fixes the issue.
```suggestion
				cfg = &config.RouteEmitterConfig{}
				cfg.JitterFactor = 0.2
```